### ISSUE TITLE
Stop fixing Translatewiki PRs manually

### ISSUE THREAD

**Issue by kelson42** [kiwix/kiwix-android#2922]
This is not substainable, not efficient, not robust, etc... see #2921

A solution should be found to avoid to tweak these PRs manually

**Comment by MohitMaliFtechiz** [comment]
hi @kelson42 .

Can you elaborate on this issue  ?

**Comment by kelson42** [comment]
@MohitMaliFtechiz Not really, I just see that you always have to fix manually things on the PR coming from Translatewiki. This is the only project where we have to do so. This is not normal, something is wrong somewhere. Would you be able here to list the problem you face when you review these PRs?

**Comment by MohitMaliFtechiz** [comment]
hi @kelson42 ,

There is some compatibility issue with string values ```tarask``` , it's does not compile without appending **old suffix** and there are some string format issue and string count error.

**Comment by kelson42** [comment]
@MohitMaliFtechiz We have only a problem with Tarask?!! Can you please be more explicit about what has to be changed each time?

**Comment by MohitMaliFtechiz** [comment]
hi @kelson42 ,

a) The very first error is

```
ERROR:kiwix-android/app/build/intermediates/merged_res/debug/values-b+be+tarask_values-b+be+tarask.arsc.flat: AAPT: error: failed to deserialize resource table: configuration has invalid locale 'be-'.
```
b) The Second error is ```Invalid format string``` in core/values-it/string.xml
```
   <string name="pref_text_zoom_summary">Aumenta o diminuisci la dimensione del testo del 25% alla volta.</string>
   ```
   This lint warning checks for two related problems:
(1) Formatting strings that are invalid, meaning that String.format will throw exceptions at runtime when attempting to use the format string.
(2) Strings containing '%' that are not formatting strings getting passed to a String.format call. In this case the '%' will need to be escaped as '%%'.

c) The Third error is ```Spelling error``` in core/values-nb/string.xml
```
   <string name="save_media">Lagre media som…</string>
   string name="save_media_saved" fuzzy="true">Lagret media som %1$s i din bildemappe.</string>
 ```

   on alphabate ```media```. It needs to replaced by ```medier``` or add ``` tools:ignore="Typos"``` annotation.
   
d) Fourth error is ```Ellipsis string can be replaced with ellipsis character``` in core/values-fr/string.xml , core/values-sl/string.xml

```
   <string name="crash_title">Hé bien... c’est embarrassant.</string> 
   <string name="preparing_files">Priprava datotek za prenos ...</string>  
   string name="performing_handshake">Rokovanje ...</string>
   <string name="preparing_files">Préparation des fichiers pour le transfert...</string>
   <string name="performing_handshake">Négociation de la liaison...</string>
```
   
   on ```...```. It needs to replace by ellipsis character ```…```
   
e) Fifth error is ```StringFormatCount```
   
   core/values-de/string.xml 
```
   <string name="pref_text_zoom_summary">Schriftgröße in 25%-Schritten ändern.</string>
```   
   core/values-nqo/string.xml
```
   <string name="pref_text_zoom_summary">ߞߟߏߘߋ߲ ߢߊ߲ߞߊ߲ ߡߊߝߊ߬ߟߋ߲߫ ߡߞߊ߬ߝߏ߬ߟߌ ٪߂߅ ߘߌ߫.</string>
```   
   Inconsistent number of arguments in formatting string pref_text_zoom_summary; found both 1 in values-de and 0 in values-nqo/strings.xml
   
It needs to add ```tools:ignore="StringFormatCount"``` in core/values-de/string.xml

**Comment by kelson42** [comment]
@MohitMaliFtechiz Thank you. Please open one ticket per type of error and provide an analysis about the cause and the possible remediations for each type error. Then close this ticket.
